components/aws/sagemaker/common/_utils.py

```diff
@@ -34,6 +39,9 @@
 # this error message is used in integration tests
 CW_ERROR_MESSAGE = 'Error in fetching CloudWatch logs for SageMaker job'
 
+# Name of session when user assumes a role
+ASSUME_SESSION_NAME_FORMAT = "kfp-sagemaker-assumed-role"
```

Comment:
does this need to be global variable ?

Reply:
1. Not really, but I wanted to avoid a magic string in the function in case we need to change it later

2. I thought about this and I'm not sure. I don't know how often people take notice of this. Here is the documentation for it - https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html . It would be another field if we want to allow the user to customize it. Maybe we should wait for a user to ask.

Reply:
Agree, Let's not add too many customizable options.

Reply:
Agreed